test/tests/functional/pbs_cray_smoketest.py

```diff
@@ -157,19 +158,17 @@ def test_cray_compute_job(self):
                            id=vname)
         # Sleep for some time before aprun actually starts
         # using the reservation
-        time.sleep(2)
+        time.sleep(6)
 
         cu = CrayUtils()
-        # Check if number of compute nodes in use are 1
+        # Check if number of compute nodes in use is 1
         self.assertEqual(cu.count_node_in_state('resv'), 1)
         if self.du.get_platform() == 'cray':
             # Cray simulator will not show anything in 'use' because
             # aprun command is just a pass through on simulator
             self.assertEqual(cu.count_node_in_state('use'), 1)
         # verify the contents of output/error files
         self.server.expect(JOB, {'job_state': 'F'}, id=jid1, extend='x')
-        # sleep a second for the files to be copied
-        time.sleep(1)
```

Comment:
You removed the sleep.  Is that because we can be assured the files have been copied out since the job is in a finished (F state)?